### ISSUE TITLE
Not/Badly code-signed binary for macOS

### ISSUE THREAD

**Issue by jcgter777** [CE-Programming/CEmu#152]
I tried to run it on my Mac, but the locked security settings don't allow this type of file. It's marked as from an unidentified developer, so it won't work.

**Comment by jacobly0** [comment]
Google says to open CEmu from the context menu in Finder (not another program).

**Comment by adriweb** [comment]
Yeah, try Right-clicking CEmu > Open.
That said, that might not work under specific security settings...

It's a pain to sign Qt apps :(

**Comment by adriweb** [comment]
@jcgter777 did it work?

When we actually have some kind of proper stable github-official-release, I'll try to work on actual signing for the binaries...

**Comment by jcgter777** [comment]
It didn't work, but thanks for helping.

**Comment by jcgter777** [comment]
@adriweb it let me open, but it instantly "quit unexpectedly".

**Comment by adriweb** [comment]
Note to self: take a close look at https://github.com/WizTeam/WizQTClient/blob/master/mac-package-qt-5-appstore.sh

**Comment by adriweb** [comment]
I'll attempt to fix this for good this weekend for the v1.0 release, however last time I tried, it didn't work, and I don't believe things have changed since then...

So hmm, if I do manage to have something working by Sunday, it's all good, otherwise I'll distribute the app as-is, with a warning somewhere on how to open it (basically just right-click > open) ... and in the meantime I'll try to fix the signing issues and eventually update the binaries in the release, so people would just have to re-download it if needed.

In conclusion, for now, in order not to postpone the release any later, I'll just move the target of this issue to "any version", and hopefully I can solve that ASAP.

**Comment by jcgter777** [comment]
@adriweb Good luck!

**Comment by adriweb** [comment]
Can someone try this build? https://usercontent.irccloud-cdn.com/file/Zhbluhfj/CEmu-mac.dmg

It's a code-signed release build, but also has some debug symbols. In addition, it's built with (technically...) the flags to be 10.8-compatible, although I have no way to try so I don't know if it works.

**Comment by jcgter777** [comment]
@adriweb Trying it!

**Comment by mateoconlechuga** [comment]
His mac OS is 5 years old. Why are we even supporting this.

**Comment by jcgter777** [comment]
@mateoconlechuga 
Why are you so salty? I know I need to update. I've yelled at my mom to do it. She needs to first backup her stuff, but she doesn't have time. I can't do it after school because there's no time. The weekends are the same. I get notifications every day asking to update. I need admin privileges to do that, and my mom won't let me because she might lose important stuff. 

@adriweb Thanks! I'm trying it!

**Comment by adriweb** [comment]
Alright, how about this one: [mac_CEmu.zip](https://github.com/CE-Programming/CEmu/files/1710658/mac_CEmu.zip)

**Comment by jcgter777** [comment]
I'm trying!

**Comment by mateoconlechuga** [comment]
It works for me :) I take it is not signed though?

**Comment by jcgter777** [comment]
It works for me as well! Thanks @adriweb !!!

**Comment by adriweb** [comment]
Alright, good. So, building and deploying with Qt 5.8 it will be... for now. (also don't recompress the .dmg, making a .zip of the (codesigned) app is good enough).
See bbf224e for a flag that should also help for compat.

**Comment by jcgter777** [comment]
:thumbs_up:
